docs/source/manual/configuration.rst

```diff
@@ -767,6 +767,126 @@ Name                   Default      Description
 type                   REQUIRED     The filter type.
 ====================== ===========  ================
 
+.. _man-configuration-json-layout:
+
+JSON layout
```

Comment:
I think it would help to see some examples of the json produced in the docs.

Reply:
Right! I will add some examples from the commit.